src/webgl/camera.js

```diff
@@ -287,28 +278,72 @@ p5.RendererGL.prototype.ortho = function(left, right, bottom, top, near, far) {
   if (top === undefined) top = +this.height / 2;
   if (near === undefined) near = 0;
   if (far === undefined) far = Math.max(this.width, this.height);
-  this.uPMatrix = p5.Matrix.identity(this._pInst);
-  //this.uPMatrix.ortho(left,right,bottom,top,near,far);
 
-  var w = right - left;
-  var h = top - bottom;
-  var d = far - near;
+  this.uPMatrix.ortho(left, right, bottom, top, near, far);
 
-  var x = +2.0 / w;
-  var y = +2.0 / h;
-  var z = -2.0 / d;
+  this._curCamera = 'custom';
+};
 
-  var tx = -(right + left) / w;
-  var ty = -(top + bottom) / h;
-  var tz = -(far + near) / d;
+/**
+ * Setup frustum camera
+ * @method  frustum
+ * @param  {Number} [left]   camera frustum left plane
+ * @param  {Number} [right]  camera frustum right plane
+ * @param  {Number} [bottom] camera frustum bottom plane
+ * @param  {Number} [top]    camera frustum top plane
+ * @param  {Number} [near]   camera frustum near plane
+ * @param  {Number} [far]    camera frustum far plane
+ * @chainable
+ * @example
+ * <div>
+ * <code>
+ * //drag mouse to toggle the world!
+ * //there's no vanish point
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ *   frustum(-1, 1, 1, -1, 1, 10000);
+ * }
+ * function draw() {
+ *   background(200);
+ *   orbitControl();
+ *   strokeWeight(0.1);
+ *   for (var i = -1; i < 2; i++) {
+ *     for (var j = -2; j < 3; j++) {
+ *       push();
+ *       translate(i * 160, 0, j * 160);
+ *       box(40, 40, 40);
+ *       pop();
+ *     }
+ *   }
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * 3 3d boxes, reveal several more boxes on 3d plane when mouse used to toggle
+ *
+ */
+p5.prototype.frustum = function() {
+  this._renderer.frustum.apply(this._renderer, arguments);
+  return this;
+};
```

Comment:
Cool! I'll merge this, but in the future please separate PRs for a new feature like this one. It doesn't seem to be an optimization, from what I can tell.